extensions/ql-vscode/src/view/common/Alert.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+type ContainerProps = {
+  type: 'warning' | 'error';
+  inverse?: boolean;
+};
+
+const getBackgroundColor = ({ type, inverse }: ContainerProps): string => {
+  if (!inverse) {
+    return 'var(--vscode-notifications-background)';
```

Comment:
```suggestion
    return 'var(--vscode-editor-background)';
```
Changes from #252526 to #1e1e1e

Reply:
I think this change deviates quite a lot from the design. The design specifies that the alerts have a different background color from the editor background and this would completely remove that. I think `notifications-background` does make sense for this, since we are essentially showing a notification inline.

Reply:
Good point. I had not considered that it needs to sit on top of an editor background. And you're right it is a notification.